src/runtime/HalideRuntimeCuda.h

```diff
@@ -1,8 +1,12 @@
 #ifndef HALIDE_HALIDERUNTIMECUDA_H
 #define HALIDE_HALIDERUNTIMECUDA_H
 
+#ifndef HALIDE_HALIDERUNTIME_H
+
```

Comment:
IMHO we'd be better off without the blank lines before and after, but I won't withhold approval because of it

Reply:
clang-format wanted them there, so I left them there. I agree they're ugly, though. Maybe there's a switch we can put in .clang-format